graphene_django/forms/tests/test_mutation.py

```diff
@@ -1,51 +1,57 @@
+import pytest
 from django import forms
-from django.test import TestCase
 from django.core.exceptions import ValidationError
 from py.test import raises
 
-from graphene import ObjectType, Schema, String, Field
+from graphene import Field, ObjectType, Schema, String
 from graphene_django import DjangoObjectType
 from graphene_django.tests.models import Film, Pet
 
 from ...settings import graphene_settings
 from ..mutation import DjangoFormMutation, DjangoModelFormMutation
 
 
-class MyForm(forms.Form):
-    text = forms.CharField()
+@pytest.fixture()
+def my_form():
+    class MyForm(forms.Form):
```

Comment:
Why do the forms need to be fixtures?

Reply:
@jkimbo I think only the types have to be fixtures, because they get registered in the global registry through some magic when python executes the class. I put everything into fixtures just for consistency.

If you like I can check what needs to be fixtures and only more these to fixtures.

Reply:
I think that would be cleaner. As far as I'm aware only the Django models and Graphene-Django DjangoObjectType's register themselves into a registry.